ldi-core/ldes-client-v2/tree-node-supplier/src/test/java/ldes/client/treenodesupplier/ProcessorTest.java

```diff
@@ -10,7 +12,8 @@ void test() {
 		Processor processor = new Processor(
```

Comment:
Again I might be missing context.. but Seeing the name "Processor" is very vague to me.. If this is a specific processor of the Client to bring it all together.. why not call it LdesClientProcessor? or ClientProcessor?